### ISSUE TITLE
[BUG] MediaElement Notification Prompt Overrides and Alters Push Notification Settings on Android 

### ISSUE THREAD

**Issue by vikher** [CommunityToolkit/Maui#2364]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [x] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

The MediaElement component triggers a notification prompt asking for permission to send notifications, even when the user is only trying to play a video.
The prompt causes confusion, as selecting "Allow" or "Don't Allow" unintentionally modifies the push notification settings, which is unrelated to video playback.
![Image](https://github.com/user-attachments/assets/68587cbb-cc73-4304-a186-a50861ef01b2)

### Expected Behavior

The MediaElement should not trigger a notification prompt if it is unrelated to video playback.
The notification prompt should not affect or change the notification settings in any way.
There should be a way to customize or disable this prompt for better user experience.

### Steps To Reproduce

1. Start the app and navigate to a page with a MediaElement component (video player).
2. Attempt to play a video.
3. The notification prompt appears, asking for permission to send notifications.
4. Selecting either "Allow" or "Don't Allow" affects push notification settings, which could confuse the user.

### Link to public reproduction project repository

https://github.com/CommunityToolkit/Maui/tree/main/samples

### Environment

```markdown
- .NET MAUI CommunityToolkit:9.1.0
- OS:Android
- .NET MAUI:8.9.92
```

### Anything else?

references the existing discussion on GitHub to provide more context and indicate that no attention has been given to the issue so far. This should help bring awareness to the problem and possibly generate more responses or solutions.
https://github.com/CommunityToolkit/Maui/discussions/2074

**Comment by ne0rrmatrix** [comment]
Media Element uses notifications to put the controls in the notification tray. If you can upvote #2074 I would appreciate it. I will bring it up at the next monthly toolkit meeting. It is a good idea and we should implement  a way to disable requesting notifications so developers can handle it directly. ATM we are blocked from new PR being commited to main until we merge #2215 and then we need to merge #2075 which is currently blocked by https://github.com/dotnet/android/issues/9535 which we have to wait for new version of dotnet android to be pushed to visual studio stable.

We can talk about updating media element to support this and I will work on a PR to fix it and add support to allow developers to choose whether to disable notifications and/or add support and control it themselves. Disabling notifications or not having support when media element is running will disable support for both lockscreen and notification controls. This will also prevent the service from running but will not impact media element running directly. Upvoting the discussion shows us how many ppl are interested in a feature and will allow me to show strong support if it has enough upvotes. I will try and push it through either way as I believe it should be done. I don't see a downside.
We will need to update the docs to reflect the changes if we do this too.

We are currently working on merging a PR that switches us from using deprecated android libraries and using media 3 instead of older libs.

**Comment by ne0rrmatrix** [comment]
All notifications can now be handled by developer and media element will not require any requests to user for access to notifications. That will be handled by OS silently with media 3 PR from now on. So this issue can be closed after today meeting. If you can test against https://github.com/CommunityToolkit/Maui/pull/2076 you can see the changes in action. The developer still needs to put the lines for service module but if they fail to do so it will crash app. It will now just not display media notifications if service is not included in android manifest. So if you want to disable media notifications u can just not include the service description in the manifest. Users will no longer be prompted at all my media element for any notification access. This is now in current PR listed above and I am requesting anyone with this issue to test against it.

**Comment by mr5z** [comment]
> Media Element uses notifications to put the controls in the notification tray.

I don't think that is true. Notifications for transport controls are typically tied to a Foreground Service when media playback occurs in the background. These notifications do not require explicit notification permissions, as they are classified as essential system notifications.

**Comment by grabnerM** [comment]
> Users will no longer be prompted at all my media element for any notification access. This is now in current PR listed above and I am requesting anyone with this issue to test against it.

Should this work by now? I just switched my entire project to .NET 9 so I can test the latest version of CommunityToolkit + MediaElement. Unfortunately, the notification request is still displayed as soon as a video is played.
